Scripts/Runtime/Entities/TaskSystem/Job/JobConfigFactory.cs

```diff
@@ -41,7 +41,7 @@ internal static class JobConfigFactory
         public static CancelJobConfig<TInstance> CreateCancelJobConfig<TInstance>(TaskFlowGraph taskFlowGraph,
                                                                                   AbstractTaskSystem taskSystem,
                                                                                   AbstractTaskDriver taskDriver,
-                                                                                  CancellableTaskStream<TInstance> taskStream,
+                                                                                  TaskStream<TInstance> taskStream,
```

Comment:
Same, do you need to assert that the provided is cancellable?

Reply:
Not here but done in `CancelJobConfig`